src/bsoncxx/array/element.hpp

```diff
@@ -27,7 +27,13 @@ BSONCXX_INLINE_NAMESPACE_BEGIN
 namespace array {
 
 ///
-/// @todo document this class
+/// A variant view type that accesses values in serialized BSON arrays.
+///
+/// Element functions as a variant type, where the kind of the element can be
+/// interrogated by calling type() and a specific value can be extracted through
+/// get_X() accessors.
+///
+/// @relatesalso document::element
```

Comment:
Actually maybe I still don't understand, isn't this tag meant to relate functions to classes, and not to relate classes to other classes? https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdrelatesalso